src/main/kotlin/Main.kt

```diff
@@ -1,31 +1,23 @@
+import kotlinx.cli.*
 import kotlinx.cli.ArgParser
 import kotlinx.cli.ArgType
 import kotlinx.cli.required
 import kotlinx.coroutines.runBlocking
 
-
-fun main(args: Array<String>) = runBlocking {
-    val parser = ArgParser("mempool-cli")
-    val startHeight by parser.option(ArgType.Int, shortName = "s", description = "Start height").required()
-
+fun main(args: Array<String>) = runBlocking() {
+    val parser = ArgParser("kotlin-cli")
+    val startHeight by parser.option(ArgType.Int, shortName = "s", description = "Start Height").required()
```

Comment:
Right now, you only handle one command, but you do it implicitly. I would like the cli to be used like this:

```sh
mempoolCli fetchTxIds --blockHeight 100000
```

Can you update the code to make it work like this? What is interesting is that this will force you to refactor to potentially handle multiple commands. For example, in the future we may also want to be able to do `mempoolCli fetchBlockId --blockHeight 100000` which is a different operation.